_data/simple-icons.json

```diff
@@ -14165,6 +14165,12 @@
             "hex": "0A84FF",
             "source": "https://design.thunderbird.net/resources/logo"
         },
+        {
+            "title": "Thunderstore",
+            "hex": "23FFB0",
+            "source": "https://github.com/thunderstore-io/brand-guidelines/blob/77c507c73902d85714e07da5e8891c9965048c81/assets/thunderstore-logomark-black.svg",
```

Comment:
```suggestion
            "source": "https://github.com/thunderstore-io/brand-guidelines/blob/master/assets/logo/thunderstore-logomark-black.svg",
```

Reply:
From your contributing guidelines:
> GitHub: For an SVG from a GitHub (GitLab, BitBucket, etc.) repository the source URL should link to the file that was used as source material. If the color does not come from the file, its origin should be stated in the Pull Request description.
\
The commit hash should always be part of the URL. On GitHub, you can get the correct URL by pressing y on the GitHub page you want to link to. You can get help at the [getting permanent links to files page](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files).

Having the commit hash in the source URL means that if the source material is updated, it is obvious which version of the image was originally used to create the `simple-icons` asset and therefore whether or not the asset here needs to be updated.

Are you sure you want me to go ahead with this amendment?

Reply:
Oh, I see - the repository path of the file has changed, which complicates things. 

Since the image contains differences, I'll re-make it from scratch using the new path.

Reply:
Sorry, my bad; I copied and pasted the wrong URL 🤦🏼‍♂️ I meant to suggest the path change only.

Reply:
Okay, cool! Glad we're on the same page :D